### ISSUE TITLE
Stringer removes the comments

### ISSUE THREAD

**Issue by To1ne** [pjaspers/stringer#5]
Output of `genstrings`:

    /* Login button */
    "login.login" = "login.login";

    /* Password label */
    "login.password" = "login.password";

    /* Register button */
    "login.register" = "login.register";

    /* Login screen title */
    "login.title" = "login.title";

    /* Username label */
    "login.username" = "login.username";

Output of `stringer`:

    /* Generated */
    "login.login" = "Login";
    "login.password" = "Password";
    "login.register" = "Register";
    "login.username" = "Username";

**Comment by pjaspers** [comment]
Yep, that's basically how I wanted it to behave, you want to keep them?

**Comment by To1ne** [comment]
So you do not type comments?

**Comment by pjaspers** [comment]
I try to give the keys meaningful enough names so they don't need comments. If the key needs commenting it's usually not that good of a key imho, but convince me otherwise :)
